test/run-local-e2e.sh

```diff
@@ -100,10 +100,10 @@ if [ -n "$E2E_FOLDER" ]; then
     log_info "Running BATS E2E tests..."
     case "$TEST_TYPE" in
     single-l2-network-fork12-op-succinct)
-        bats ./tests/aggkit/bridge-e2e.bats ./tests/aggkit/e2e-pp.bats ./tests/aggkit/bridge-sovereign-chain-e2e.bats
+        bats ./tests/aggkit/bridge-e2e.bats ./tests/aggkit/e2e-pp.bats ./tests/aggkit/bridge-sovereign-chain-e2e.bats ./tests/aggkit/claim-call-data.bats
         ;;
     single-l2-network-fork12-pessimistic)
-        bats ./tests/aggkit/bridge-e2e-custom-gas.bats ./tests/aggkit/bridge-e2e.bats ./tests/aggkit/e2e-pp.bats
+        bats ./tests/aggkit/bridge-e2e-custom-gas.bats ./tests/aggkit/bridge-e2e.bats ./tests/aggkit/e2e-pp.bats ./tests/aggkit/claim-call-data.bats
         ;;
```

Comment:
Nitpick, let's wrap the commands, it makes it slightly more readable IMO. WDYT?
```suggestion
    case "$TEST_TYPE" in
    single-l2-network-fork12-op-succinct)
        bats \
            ./tests/aggkit/bridge-e2e.bats \
            ./tests/aggkit/e2e-pp.bats \
            ./tests/aggkit/bridge-sovereign-chain-e2e.bats \
            ./tests/aggkit/claim-call-data.bats
        ;;
    single-l2-network-fork12-pessimistic)
        bats \
            ./tests/aggkit/bridge-e2e-custom-gas.bats \
            ./tests/aggkit/bridge-e2e.bats \
            ./tests/aggkit/e2e-pp.bats \
            ./tests/aggkit/claim-call-data.bats
        ;;
```